docs/Overview.md

```diff
@@ -194,27 +206,26 @@ The IS-05 schemas `sender_transport_params_ndi.json` and `constraints_schema_sen
 ```json
 [
   {
-    "server_ip": "10.10.10.10",
-    "server_port": 5960,
+    "machine_name": "ndi-machine-name",
     "source_name": "ndi-sender-unique-name",
-    "group_name": "camera1"
+    "source_url" : "...",
+    "source_address" : "10.10.10.123:5906"
   }
 ]
 ```
+#### machine_name
+The name of the sender node in the NDI domain. A node MUST report the machine name being utilized by the NDI SDK. A controller SHOULD NOT specify a new `machine_name` but SHOULD use `null` or the current `machine_name`. 
```

Comment:
"but SHOULD use **`auto`** or the current `machine_name`."

I think `auto` should be used to tell the Sender to select the proper value.